lib/sanitize.js

```diff
@@ -0,0 +1,25 @@
+var dompurify = require('dompurify');
+
+
+const allowedUriSchemesRegexp = /^(https?|mailto|#)/i;
```

Comment:
Just to keep the codebase consistent, going with ES5 all around:
```suggestion
var allowedUriSchemesRegexp = /^(https?|mailto|#)/i;
```

Reply:
changed